frontend/src/metabase/dashboard/components/ParametersPopover.tsx

```diff
@@ -63,20 +53,21 @@ const ParameterOptionsSection = ({
   section: ParameterSection;
   onClick: () => void;
 }) => (
-  <OptionItemRoot onClick={onClick}>
-    <OptionItemTitle
-      className={cx(CS.textBold, CS.flex, CS.alignCenter)}
-      style={{ marginBottom: 4 }}
-    >
+  <Box className={S.row} display="table-row" onClick={onClick}>
```

Comment:
We cannot use `grid` here because we need to hover the entire row. And `display: contents` isn't supported super well. 
We cannot use `flex` here without hard-coding column widths.